testDijkstra (__main__.TestAlgo) ... FAIL
testIndiceMinimum (__main__.TestAlgo) ... ok
testNoeudMinimalNonVisitesDeNoeudA (__main__.TestAlgo) ... FAIL
testNoeudMinimalNonVisitesDeNoeudB (__main__.TestAlgo) ... FAIL
testNoeudsVoisinsA (__main__.TestAlgo) ... FAIL
testNoeudsVoisinsB (__main__.TestAlgo) ... FAIL
testnoeudMinimalNonVisites (__main__.TestAlgo) ... FAIL
testCarteEnSommets (__main__.TestCarte) ... FAIL
testDistance3D (__main__.TestCarte) ... ok
testDistanceEuclidienne (__main__.TestCarte) ... ok
testEstVoisinA (__main__.TestCarte) ... ok
testEstVoisinB (__main__.TestCarte) ... ok
testMatriceAdjacence (__main__.TestCarte) ... FAIL
testChargeMatriceDeCarte (__main__.TestFichier) ... FAIL
testLireCarte (__main__.TestFichier) ... ERROR
testAfficheChemin (__main__.TestInterface) ... FAIL
testGenereMatriceAleatoireA (__main__.TestInterface) ... FAIL
testGenereMatriceAleatoireB (__main__.TestInterface) ... ERROR
testSaisirMatrice (__main__.TestInterface) ... FAIL

======================================================================
ERROR: testLireCarte (__main__.TestFichier)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "c:\Users\bough\Documents\GitHub\2021A_TP3\test_assignment.py", line 150, in testLireCarte
    carte = lireCarte(nomFichier)
  File "c:\Users\bough\Documents\GitHub\2021A_TP3\Fichier.py", line 7, in lireCarte
    lire = open("carte.txt", "r")
FileNotFoundError: [Errno 2] No such file or directory: 'carte.txt'

======================================================================
ERROR: testGenereMatriceAleatoireB (__main__.TestInterface)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "c:\Users\bough\Documents\GitHub\2021A_TP3\test_assignment.py", line 172, in testGenereMatriceAleatoireB
    for i, ligne in enumerate(matrice):
TypeError: 'ellipsis' object is not iterable

======================================================================
FAIL: testDijkstra (__main__.TestAlgo)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "c:\Users\bough\Documents\GitHub\2021A_TP3\test_assignment.py", line 108, in testDijkstra
    self.assertTupleEqual(tuple_1, tuple_2)
AssertionError: First sequence is not a tuple: Ellipsis

======================================================================
FAIL: testNoeudMinimalNonVisitesDeNoeudA (__main__.TestAlgo)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "c:\Users\bough\Documents\GitHub\2021A_TP3\test_assignment.py", line 70, in testNoeudMinimalNonVisitesDeNoeudA
    self.assertTupleEqual(tuple_1, tuple_2)
AssertionError: First sequence is not a tuple: Ellipsis

======================================================================
FAIL: testNoeudMinimalNonVisitesDeNoeudB (__main__.TestAlgo)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "c:\Users\bough\Documents\GitHub\2021A_TP3\test_assignment.py", line 78, in testNoeudMinimalNonVisitesDeNoeudB
    self.assertTupleEqual(tuple_1, tuple_2)
AssertionError: First sequence is not a tuple: Ellipsis

======================================================================
FAIL: testNoeudsVoisinsA (__main__.TestAlgo)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "c:\Users\bough\Documents\GitHub\2021A_TP3\test_assignment.py", line 93, in testNoeudsVoisinsA
    self.assertTupleEqual(tuple_1, tuple_2)
AssertionError: First sequence is not a tuple: Ellipsis

======================================================================
FAIL: testNoeudsVoisinsB (__main__.TestAlgo)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "c:\Users\bough\Documents\GitHub\2021A_TP3\test_assignment.py", line 100, in testNoeudsVoisinsB
    self.assertTupleEqual(tuple_1, tuple_2)
AssertionError: First sequence is not a tuple: Ellipsis

======================================================================
FAIL: testnoeudMinimalNonVisites (__main__.TestAlgo)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "c:\Users\bough\Documents\GitHub\2021A_TP3\test_assignment.py", line 85, in testnoeudMinimalNonVisites
    self.assertTupleEqual(tuple_1, tuple_2)
AssertionError: First sequence is not a tuple: Ellipsis

======================================================================
FAIL: testCarteEnSommets (__main__.TestCarte)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "c:\Users\bough\Documents\GitHub\2021A_TP3\test_assignment.py", line 144, in testCarteEnSommets
    self.assertDictEqual(sommets, carte_data_ex["sommets"])
AssertionError: Ellipsis is not an instance of <class 'dict'> : First argument is not a dictionary

======================================================================
FAIL: testMatriceAdjacence (__main__.TestCarte)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "c:\Users\bough\Documents\GitHub\2021A_TP3\test_assignment.py", line 139, in testMatriceAdjacence
    self.assertListEqual(matrice_1, matrice_2)
AssertionError: First sequence is not a list: Ellipsis

======================================================================
FAIL: testChargeMatriceDeCarte (__main__.TestFichier)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "c:\Users\bough\Documents\GitHub\2021A_TP3\test_assignment.py", line 156, in testChargeMatriceDeCarte
    self.assertListEqual(carte, fichier_data_ex["carte_2"])
AssertionError: First sequence is not a list: Ellipsis

======================================================================
FAIL: testAfficheChemin (__main__.TestInterface)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "c:\Users\bough\Documents\GitHub\2021A_TP3\test_assignment.py", line 187, in testAfficheChemin
    self.assertMultiLineEqual(interface_data_ex["txt"], txt)
AssertionError: Ellipsis is not an instance of <class 'str'> : Second argument is not a string

======================================================================
FAIL: testGenereMatriceAleatoireA (__main__.TestInterface)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "c:\Users\bough\Documents\GitHub\2021A_TP3\test_assignment.py", line 166, in testGenereMatriceAleatoireA
    self.assertTrue(isinstance(genereMatriceAleatoire(5), list))
AssertionError: False is not true

======================================================================
FAIL: testSaisirMatrice (__main__.TestInterface)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\bough\AppData\Local\Programs\Python\Python39\lib\unittest\mock.py", line 1337, in patched
    return func(*newargs, **newkeywargs)
  File "c:\Users\bough\Documents\GitHub\2021A_TP3\test_assignment.py", line 163, in testSaisirMatrice
    self.assertListEqual(saisirMatrice(), matrice)
AssertionError: First sequence is not a list: Ellipsis

----------------------------------------------------------------------
Ran 19 tests in 0.005s

FAILED (failures=12, errors=2)
